src/wet-lab/design/solid-phase.md

```diff
@@ -1 +1,51 @@
 # Solid phase DNA Synthesis
+
+## Solid phase DNA Synthesis
+
+This document outlines the process of non-traditional solid phase DNA synthesis through the use of tdt. In particular, this document will provide a broad overview of the workflow and the details of primer immobilization, solid phase chemistry, and primer cleavage. It also outlines the chemistry of monomers intended to use.
+Context and Scope
+Traditional solid phase chemical synthesis occurs from 3’ -> 5’ as shown below.
+![chemical solid phase DNA synthesis](Chemical-SPPS.png)
+
+This is an important consideration as a robust set of chemical reactions exist for the immobilization of nucleic acids on the solid phase. However, there is a comparatively smaller tool box for synthesis in the reverse (5’->3’) direction. Additionally, chemical solid phase synthesis involves protecting groups on the exocyclic nitrogenous bases. For our purposes, we require a method to immobilize the 5’ end on the solid phase, leaving the 3’ end open to react with incoming DNTPs.
+
+For controlled single nucleotide insertion, a robust yet readily cleavable protecting group is required. The primary method of 3’ protection for 5’ to 3’ synthesis is photolabile protecting groups. These groups are efficiently cleaved when exposed to UV light. These groups have been developed due to the desire to generate complex microarrays for genetic analysis. With photolabile protecting groups, a desired pattern of unique oligonucleotides may be synthesized. This chemistry can be leveraged for the use of TDT single nucleotide insertion.
+
+## Goals
+The goal of this phase is 
+  1. To functionalize the surface for the immobilization of primers
+  2. To immobilize the primer 5’ end on the solid phase at a high enough density and selectivity to enable efficient DNA synthesis
+  3. To characterize the density of primer immobilization
+  4. Lastly, the primer and newly synthesized DNA strand must be cleaved from the solid phase
+
+## The actual design of immobilization
+
+There are two different methods being considered for immobilization. All approaches are based on the same principle. 
+
+Disulfide 5’ modifier and dibenzocylooctyne 5’ modifier are the primer methods in consideration. The idea with each modified primer is to use orthogonal reacting functional groups. As a DNA oligo has both a 3’ and 5’ free hydroxyl group, using a 5’ modifier that selectively reacts with its counterpart would prevent the 3’ hydroxyl from reacting in any meaningful quantities.
+![Sulfide solid phase synthesis](Sulfide-SPS.png)
```

Comment:
For images, add a new line before and after the image so the image is on it's own line and there isn't text beside the image. 

Refer to previous comment about images.